setup.py

```diff
@@ -0,0 +1,34 @@
+"""Setup file for using congress as a python package."""
+from os import path
+
+from setuptools import setup
+
+# Obtain long_description from README.md
+here = path.abspath(path.dirname(__file__))
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='united-states-congress',
+    version='0.0.1',
+    author='Sunlight Foundation',
```

Comment:
This project was created collaboratively by several organizations, and the Sunlight Foundation no longer exists. Credit should be to "The unitedstates organization on GitHub".

Reply:
sounds good I'll change it rq

Reply:
Fixed